### ISSUE TITLE
Modules missing when installing with conda

### ISSUE THREAD

**Issue by jkosinski** [cctbx/cctbx_project#740]
Hi,

When I install with `conda create -n my_env -c conda-forge cctbx-base`, modules `probe`, `reduce`, and `chem_data` are not being installed. Is there a way of installing them in the environment?

Best,
Jan

**Comment by bkpoon** [comment]
For `probe` and `reduce`, we are migrating to the `probe` and `reduce` code inside `mmtbx`. These are the `mmtbx.probe2` and `mmtbx.reduce2` commands. The work and testing is being done by the MolProbity folks at Duke (@chrisjwilliams).

The `chem_data` repository will not be included in the conda package because it is about 2 GB. We are in the planning stages of cleaning up and hosting `chem_data` in a public repository and then you could use an environment variable to point to that location or make a symbolic link to the directory inside your conda environment. We'll probably add a command to simplify this process.

If you have a Phenix installation, you can make a symbolic link to the path to `chem_data` in `${CONDA_PREFIX}/lib/python3.10/site-packages`. That is for Python 3.10. Replace the `python3.10` part with the version of Python in your active conda environment.
```
ln -s <Phenix installation>/modules/chem_data ${CONDA_PREFIX}/lib/python3.10/site-packages/
```
And then you can verify that there is a `chem_data` directory in `${CONDA_PREFIX}/lib/python3.10/site-packages/`. After that, you can use the new `mmtbx.probe2` and `smtbx.reduce2` commands. But keep in mind that they are still being tested.

**Comment by bkpoon** [comment]
One additional note about `chem_data`, there are databases stored as pickle files that should be regenerated for Python 3. So if you are using the Phenix copy of `chem_data`, you should copy that directory somewhere first and then link the copied version to the conda environment.

Inside the `chem_data` directory, there are directories named `cablam_data` and `rotarama_data`. The files ending in `.pickle` and `.dlite` should be deleted. Then run the `mmtbx.rebuild_cablam_cache` and `mmtbx.rebuild_rotarama_data` commands in your environment to rebuild the databases.

**Comment by wenchangzhou-qtx** [comment]
Hey @bkpoon,

Thanks for having such a wonderful repo! I actually have a similiar issue with my installation. After I install with conda, I cannot see mmtxb.refinement package in it, am I missing something? Thanks

**Comment by wenchangzhou-qtx** [comment]
Thanks @bkpoon I can run mmtbx.reduce but not mmtbx.refinement.

**Comment by bkpoon** [comment]
The refinement programs are in Phenix. We have `phenix.refine` for diffraction data and `phenix.real_space_refine` for fitting into maps. The underlying algorithms are in the mmtbx.refinement module (and other parts of cctbx), but the tools are part of Phenix.

**Comment by wenchangzhou-qtx** [comment]
Got it. I'm trying to use `mmtbx.reifnement` module to accomplish the same task as `phenix.refine`, is this possible within mmtbx?

**Comment by bkpoon** [comment]
Yes, it is possible, but it can get complicated if you are trying to reproduce all the functionality in `phenix.refine`. For example, `phenix.refine` refines specific things inside each macro cycle, like the coordinates or B-factors. The individual parts are in `cctbx-base`, but you would have to combine all the parts together. I would suggest that you get access to Phenix if you are trying to do something for production use.

**Comment by russell-taylor** [comment]
Even though chem_data has been moved out of Phenix, it looks like `conda.bat create -n my_env -c conda-forge cctbx-base` installs an older version of mmtbx.reduce2 and doesn't pull chem_data. Is there a new release of that planned?

**Comment by bkpoon** [comment]
@russell-taylor The `chem_data` package needs to be installed separately because it is large and not everyone needs it. The package is available on the releases page. There is an issue with this package on Windows because there are some filenames that start with `AUX`. The next release will have a package that works with Windows.

**Comment by wangjiawen2013** [comment]
> Makes sense. I modified the README.md in reduce2 and probe2 to clarify that the instructions for installing chem_data need to be followed.

Yes, I saw your reply on https://github.com/rlabduke/reduce/issues/62#issuecomment-2901176751, but I am still blocked when  installing monomer library.

In your reply, you said "You can download a .conda package from the linked-to releases page and then after activating your conda environment, run conda install -c cctbx cctbx_monomer_library to install it." while in cctbx-project's home page, it displays:

![Image](https://github.com/user-attachments/assets/fbfc8d9e-64ae-4fb2-8eb5-2294ce138bf9)

I have downloaded `chem_data-2025.4-pyh66f2a00_0.conda`, then what should I do with `chem_data-2025.4-pyh66f2a00_0.conda` ? Should I run `conda install -c cctbx cctbx_monomer_library` or `conda install <chem_data package>` or something else ?
